components/dashboard/DashboardBlocksBlock.js

```diff
@@ -28,13 +29,13 @@ const DashboardBlocksBlock = props => (
               />
               <span className="d-none d-sm-block">@{props.witness}</span>
               <span className="d-sm-none">
-                @{props.witness ? props.witness.substring(0, 7) : null}..
+                {props.witness ? `@${props.witness}` : null}..
```

Comment:
@frol I used `props.witness.substring(0, 7)` because that section is rendered only on mobiles, where anything greater than that will break the screen.

Reply:
Thank you for pointing it out; I will bring the substring call back.